tests/meson.build

```diff
@@ -89,7 +89,7 @@ if gtest_dep.found()
     install_dir: unittest_install_dir
   )
 
-  test('unittest_src_iio', unittest_src_iio, args: ['--gst-plugin-path=..'])
+  test('unittest_src_iio', unittest_src_iio, timeout: 60, args: ['--gst-plugin-path=..'])
```

Comment:
I think the theoretical max timeout should be 150 as 14 of the tests can in worst case last up to 10 sec each (Timeout set for polling the file). But 60 should practically work I think.

Reply:
I will update to 120 sec. Let's consider to change this later if we implement more testcases about tensor stream.